vm/src/builtins/list.rs

```diff
@@ -458,6 +465,7 @@ fn do_sort(
 #[derive(Debug)]
 pub struct PyListIterator {
     pub position: AtomicCell<usize>,
+    status: AtomicCell<IterStatus>,
     pub list: PyListRef,
```

Comment:
any reason why these are public currently?

Reply:
I have no idea. @coolreader18 is this a part of public API?